src/repositories/projectsRepository.ts

```diff
@@ -0,0 +1,23 @@
+import { Db } from 'mongodb';
+import { IConfig } from 'config';
+import { BaseRepository } from './baseRepository';
+import { ILogger } from '../services/logger';
+
+/**
+ * Project information from docs_metadata.projects for parser builds.
+ */
+export class ProjectsRepository extends BaseRepository {
+  constructor(db: Db, config: IConfig, logger: ILogger) {
+    super(config, logger, 'ProjectsRepository', db.collection('projects'));
+  }
+
+  async getProjectEntry(name: string): Promise<any> {
```

Comment:
If we can, it would be great to properly type the return type. I know that the types are tied to the `BaseRepository`, so it might be a good chunk of work to make that possible. If that is the case, then no worries about having to do the work of updating all the types.

Reply:
I added in a proper return type, but it's worth noting that nearly all of the methods in the `repositories` directory have a return type of `any`